src/client/components/Home/RegisterButton.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {useHistory} from 'react-router-dom';
+import Button from '../Button/Button';
+import './RegisterButton.style.css';
+
+function RegisterButton() {
+    const history = useHistory();
+
+    const redirect = () => {
+      history.push('/sign-up')
+    }
+    return (
+        <div className="homepage">
+          <section className="homepage-slogan">
+          <h2>Improve Your Resume To Present Your Best Self</h2>  
+          <p>Upload your Resume to get an evaluation</p>
+          <Button buttonName='Register/Sign Up' style={{ backgroundColor: 'orange' }} onClick={redirect}/>
```

Comment:
This color for button :#F25B06 and reverse in order of Register/Sign Up

Reply:
I think the position for the h2 and p also should be added ,, please check figma